template.yml

```diff
@@ -49,14 +49,50 @@ Parameters:
     NoEcho: true
     Description: "Buildkite Agent token from https://buildkite.com/organizations/-/agents"
 
-  BuildkiteAgentQueue:
+  BuildkiteQueue:
```

Comment:
This is a great suggestion for consistency between our elastic stack templates. However, it's not a backwards compatible change for existing consumers of the template.

Reply:
fixed. @matthewborden